addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -90,7 +90,7 @@
                     <i class="fa fa-navicon"/>
                 </button>
             </div>
-            <div class="d-flex justify-content-lg-end gap-2 flex-grow-1 mw-lg-50">
+            <div t-attf-class="d-flex justify-content-lg-end gap-2 flex-grow-1 w-100 w-lg-auto #{search and 'mw-xl-75' or 'mw-xl-50'}">
```

Comment:
Even though it works, we do not use `X and A or B` in python but `A if W else B`:
```suggestion
            <div t-attf-class="d-flex justify-content-lg-end gap-2 flex-grow-1 w-100 w-lg-auto #{'mw-xl-75' if search else 'mw-xl-50'}">
```

Reply:
Okay ! 
What about just `X and A` ? Should that become `A if X` ?

Reply:
No, you have to add ` else ''` in that case.